neutron_lbaas_dashboard/api/rest/__init__.py

```diff
@@ -22,5 +22,6 @@
 """
 
 # import REST API modules here
-from neutron_lbaas_dashboard.api.rest import barbican  # noqa
-from neutron_lbaas_dashboard.api.rest import lbaasv2  # noqa
+from f5_neutron_lbaas_dashboard.api.rest import barbican  # noqa
```

Comment:
Python library path will be changed by build script automatically. We will not need to made this change in code repo.